packages/fiori/cypress/specs/NotificationList.cy.tsx

```diff
@@ -56,6 +57,600 @@ function Sample() {
 	</NotificationList>;
 }
 
+describe("Notification List Item Tests", () => {
+	it("tests itemToggle fired", () => {
+		const stub = cy.stub().as("myStub");
+		cy.mount(<NotificationListGroupItem id="nlgi1" onClick={stub} />);
```

Comment:
Wrap this group around NotificationList and attach handler to the `item-toggle` event (onItemToggle). Delete the click handler